lib/rubocop/cop/layout/else_alignment.rb

```diff
@@ -136,6 +142,14 @@ def check_alignment(base_range, else_range)
           )
           add_offense(else_range, location: else_range, message: message)
         end
+
+        def assignment_node(node)
+          assignment_node = node.ancestors.first
+          return nil unless
+            assignment_node&.assignment?
```

Comment:
This is a small feedback. It seems to be simple :-)

```diff
-          return nil unless
-            assignment_node&.assignment?
+          return unless assignment_node&.assignment?
```

Reply:
Good catch. I have copied this method from another cop. Let me fix it.

Reply:
fixed